v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/failureinjectiontesting/SrcDbToSpannerMySQLSevereErrorShardedFT.java

```diff
@@ -155,14 +157,16 @@ public void dataflowWorkerFailureTest() {
         ChainedConditionCheck.builder(
                 List.of(
                     SpannerRowsCheck.builder(spannerResourceManager, AUTHORS_TABLE)
-                        .setMinRows(1900)
-                        .setMaxRows(1900)
+                        .setMinRows(190)
+                        .setMaxRows(190)
                         .build(),
                     SpannerRowsCheck.builder(spannerResourceManager, BOOKS_TABLE)
-                        .setMinRows(1900)
-                        .setMaxRows(1900)
+                        .setMinRows(190)
+                        .setMaxRows(190)
                         .build(),
-                    new SevereErrorsCheck(pipelineLauncher, jobInfo, 100)))
+                    DlqEventsCountCheck.builder(gcsResourceManager, "output")
```

Comment:
Would it be possible to  change this to a path to the DLQ folder within the `output` directory?
There could be more folders added in future and referring to `dlq` or `dlq/severe` folder might future proof the test case.

Reply:
Done.